### PR TITLE
HTBHF-2510 define POST endpoint on journey definition

### PR DESCRIPTION
**Pull request by gethyn1**
Allow journeys to define the endpoint when making POST request on terms and conditions page:

- Tidy unit tests for terms and conditions `post.js`
- Add unit test for claimant service POST request
- Define POST endpoint on journey definition

### CODE REVIEW

File: src/web/routes/application/journey-definitions.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const {
 
 const APPLY = {
   name: 'apply',
+  endpoint: '/v2/claims',
```
Comment (dwybourn) [inline src/web/routes/application/journey-definitions.js]: endpoint to me sounds like the endpoint to start the journey. claimsEndpoint might work better?

Comment (YetAnotherMatt) [reply src/web/routes/application/journey-definitions.js]: or `postEndpoint`?

Comment (dwybourn) [reply src/web/routes/application/journey-definitions.js]: Still sounds like an endpoint on the UI. claimServiceEndpoint or something like that. Just needs to be clear that this is the path of a down stream service at the end of the journey.

Comment (gethyn1) [reply src/web/routes/application/journey-definitions.js]: I would rather explain this in the README (there is a story for updating this). I think `endpoint` and a search for where it is used is enough .. unless you feel really strongly about this??

Comment (YetAnotherMatt) [reply src/web/routes/application/journey-definitions.js]: :man_shrugging:

**Pull request merged**